website/source/docs/providers/aws/r/config_config_rule.html.markdown

```diff
@@ -0,0 +1,135 @@
+---
+layout: "aws"
+page_title: "AWS: aws_config_config_rule"
+sidebar_current: "docs-aws-resource-config-config-rule"
+description: |-
+  Provides an AWS Config Rule.
+---
+
+# aws\_config\_config\_rule
+
+Provides an AWS Config Rule.
+
+~> **Note:** Config Rule requires an existing [Configuration Recorder](/docs/providers/aws/r/config_configuration_recorder.html) to be present. Use of `depends_on` is recommended (as shown below) to avoid race conditions.
```

Comment:
Ideally yes, but there's no `configuration_recorder` field or anything like that - the API just currently relies on the fact that you can only have a single recorder per region, so they probably think there's no need to ask which recorder this rule belongs to.

Therefore it's unfortunately necessary.